tests/admin/test_admin_users_endpoints.py

```diff
@@ -59,6 +60,62 @@ def encoded_admin_jwt(kid, rsa_private_key):
     claims["scope"].append("admin")
     return jwt.encode(claims, key=rsa_private_key, headers=headers, algorithm="RS256")
 
+@pytest.fixture(scope="function")
+def mock_arborist_request():
+    """
+    This fixture returns a function which you call to mock out arborist endpoints.
+    Give it an argument in this format:
+        {
+            "arborist/health": {
+                "GET": ("", 200)
+            },
+            "arborist/auth/request": {
+                "POST": ({"auth": False}, 403)
+            }
+        }
+    """
+
+    def do_patch(urls_to_responses=None):
+        urls_to_responses = urls_to_responses or {}
```

Comment:
@microbe73 Instead of copying and pasting / duplicating this function here import it from the other file.